src/components/dialogs/input-hooks.js

```diff
@@ -680,9 +681,18 @@ export const useExpandableValues = ({
     fieldProps,
     validateItem,
 }) => {
+    const useCostumStyle = makeStyles((theme) => ({
```

Comment:
Costum or custom ?

Reply:
my bad

Reply:
That could be costum as it's come css. I add a doubt :)